client/packages/invoices/src/Prescriptions/ListView/Toolbar.tsx

```diff
@@ -29,6 +29,12 @@ export const Toolbar: FC<{ filter: FilterController }> = () => {
               urlParameter: 'otherPartyName',
               isDefault: true,
             },
+            {
+              type: 'text',
+              name: t('label.reference'),
+              urlParameter: 'theirReference',
+              isDefault: false,
+            },
```

Comment:
I only have one small comment in that I would think that "Status" should go above "Reference" in the dropdown list, but it's not a biggie and someone else may think otherwise.